app/src/main/res/values/strings.xml

```diff
@@ -443,6 +443,11 @@ Upload your first media by tapping on the add button.</string>
   <string name="get_content_photo_picker_title">Use GET_CONTENT photo picker</string>
   <string name="get_content_photo_picker_explanation">Disable if your pictures get uploaded without location</string>
   <string name="location_loss_warning">Please make sure that this new Android picker does not strip location from your pictures.</string>
+  <string name="in_app_camera_location_access_explanation">Allow the app to fetch location in case the camera does not record it. Some device cameras do not record location. In such cases, letting the app fetch and attach location to it makes your contribution more useful.</string>
```

Comment:
It looks alright but a suggestion to change the tone of message - 

> Allow the application to retrieve location data in the event that the camera fails to capture it. Certain device cameras may not have the capability to record location information. In such circumstances, enabling the application to fetch and append location data enhances the value of your contribution.

Reply:
> Please remember that you can modify this setting at any time.

Sounds good. Will add this once we're sure if we're using a "Do not ask again" checkbox or a toggle button in the Settings.